.github/ISSUE_TEMPLATE/irs-story

```diff
@@ -0,0 +1,22 @@
+---
+name: Propose a Story
+about: propose a new story, new task
+---
+<!-- 
+A clear and concise description of what the desired story will look like. 
+-->
+**As** ... ,
+**I want** ... ,
+**so that** ... .
+
+## Hints / Details
+<!--
+provide as much information regarding this request as possible
+-->
+- ...
+
+## Outcome / Acceptance Criteria
+<!--
+describe the Outcome the result of the story
+-->
```

Comment:
One idea is to define the "##Out of scope:" section explicitly - as if we do not have a structure for this everyone places and manages the out of scope individually. This is hard to cover.